--- miniaudio.h	2025-05-21 14:18:54
+++ miniaudio.hh	2025-05-21 11:15:34
@@ -3090,5 +3090,5 @@
 Android backends will report this notification.
 */
-typedef void (* ma_device_notification_proc)(const ma_device_notification* pNotification);
+typedef void (* ma_device_notification_proc)(const void* pNotification);
 
 
@@ -3133,5 +3133,5 @@
 The proper way to stop the device is to call `ma_device_stop()` from a different thread, normally the main application thread.
 */
-typedef void (* ma_device_data_proc)(ma_device* pDevice, void* pOutput, const void* pInput, ma_uint32 frameCount);
+typedef void (* ma_device_data_proc)(void* pDevice, void* pOutput, const void* pInput, ma_uint32 frameCount);
 
 
@@ -3157,5 +3157,5 @@
 Do not restart or uninitialize the device from the callback.
 */
-typedef void (* ma_stop_proc)(ma_device* pDevice);  /* DEPRECATED. Use ma_device_notification_proc instead. */
+typedef void (* ma_stop_proc)(void* pDevice);  /* DEPRECATED. Use ma_device_notification_proc instead. */
 
 typedef enum
